pkg/ingester/api.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+package ingester
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/grafana/mimir/pkg/ingester/client"
+	"github.com/grafana/mimir/pkg/mimirpb"
+)
+
+type IngesterAPI interface {
```

Comment:
This should be just `API` since the package is already is `ingester` so the full name would be `ingester.API` instead of `ingester.IngesterAPI` (I'm surprised that no linter complained).

Reply:
> I'm surprised that no linter complained

me too